src/main/resources/translations.json

```diff
@@ -1,142 +1,143 @@
+    "es": "<p>Se acaba de autorizar su monedero!</p>\n <p>La dirección de su monedero es: {{activationData}}</p>\n <p>Puede comenzar a invertir haciendo un depósito en <a href=\"{{& link}}\">{{& link}}</p>",
+    "el": "<p>Το πορτοφόλι σου μόλις εγκρίθηκε!</p>\n<p>Το πορτοφόλι σου βρίσκεται εδώ: {{activationData}}</p>\n<p>Μπορείς να αρχίσεις να επενδύεις κάνοντας μια κατάθεση στο λογαριασμό σου <a href=\"{{& link}}\">{{& link}}</p>\n"
+  },
+  "userWalletTemplate": {
+    "en": "<p>New wallet has just been created on crowdfunding platform that you manage!</p>\n<p>New user wallet address is: {{activationData}}</p>\n<p>Please go <a href=\"{{& link}}\">{{& link}}</a> to review and approve new wallet creation.</p>",
+    "es": "<p>Se acaba de crear un nuevo monedero en la plataforma de inversión colectiva que usted gestiona!</p>\n <p>La dirección del nuevo monedero de usuario es: {{activationData}}</p>\n <p>Por favor, diríjase a <a href=\"{{& link}}\">{{& link}}</a> para revisar y autorizar la creación del nuevo monedero.</p>",
+    "el": "<p>Ένα νέο πορτοφόλι μόλις δημιουργήθηκε στην πλατφόρμα crowdfunding που διαχειρίζεσαι!</p>\n<p>Το πορτοφόλι βρίσκεται εδώ: {{activationData}}</p>\n<p>Πήγαινε στο <a href=\"{{& link}}\">{{& link}}</a> για να ελεγξεις και να εγκρίνεις τη δημιουργία του.</p>\n"
+  },
+  "withdrawRequestTemplate": {
+    "en": "<p>You have requested withdraw with amount: €{{amount}}.</p>\n\n<p>You will be informed after your withdraw request has been processed.</p>",
+    "es": "<p>Ha solicitado una retirada por un importe de: {{amount}}€.</p>\n \n <p>Se le informará cuando su solicitud de retirada se haya procesado.</p>",
+    "el": "<p>Ζήτησες ανάληψη με ποσό:{{amount}}€.</p>\n\n<p>Θα ενημερωθείς μόλις επεξεργαστούμε το αίτημά σου.</p>\n"
+  },
+  "withdrawTemplate": {
+    "en": "{{#burned}}\n    <p>Your withdraw have been approved and your funds have been transferred to your bank account.</p>\n{{/burned}}\n{{^burned}}\n    <p>Your withdraw request has been rejected. For more information contact support.</p>\n{{/burned}}",
+    "es": "{{#burned}}\n  <p>Su retirada ha sido autorizada y sus fondos se han transferido a su cuenta bancaria.</p>\n {{/burned}}\n {{^burned}}\n  <p>Su solicitud de retirada ha sido rechazada. Para más información, contacte con soporte.</p>\n {{/burned}}",
+    "el": "{{#burned}}\n    <p>Η ανάληψή σου έχει εγκριθεί και τα χρήματά έχουν μεταφερθεί στον τραπεζικό σου λογαριασμό.</p>\n{{/burned}}\n{{^burned}}\n    <p>Το αίτημα ανάληψης απορρίφθηκε. Για περισσότερες πληροφορίες επικοινώνησε μαζί μας.</p>\n{{/burned}}\n"
+  },
+  "investmentWithTosTemplate": {
+    "en": "<p>You have successfully invested amount: €{{amount}} in project {{projectName}}.</p>\n<p>Investment is completed under conditions provided in the attached file.</p>"
+  },
+  "investmentTemplate": {
+    "en": "<p>You have successfully invested amount: €{{amount}} in project {{projectName}}.</p>"
+  },
```

Comment:
This can be in a single template. Use boolean `hasTos` or something like that, see `withdrawTemplate`.